scripts/verify_cairo_listings.sh

```diff
@@ -2,4 +2,8 @@
 set -xe
 
 # TODO(#2718)
-for d in ./docs/listings/*; do (cd "$d" && scarb build); done
+for d in ./docs/listings/*; do
+    if [ -f "$d/Scarb.toml" ]; then
+        (cd "$d" && scarb build);
+    fi    
+done
```

Comment:
I wonder if there is any case where this logic becomes problematic. What if someone added a package without `Scarb.toml`? Would we be skipping tests of it or would some other test catch it?
Maybe instead of having this kind of logic here, move `example_workflows` outside of 
`listings`?